origins_workflow/src/Form/AuditSettingsForm.php

```diff
@@ -200,6 +200,15 @@ private function removeAuditField($type) {
    * Add audit field to the content type.
    */
   private function addAuditField($type) {
+    // Check to see if this is the health condition content type and the
+    // 'field_next_review_date' is present. If it is then we don't need to
+    // add 'field_next_audit_due'.
+    if ($type == 'health_condition') {
```

Comment:
I wonder if this would (eventually) sit better in the nidirect_healthconditions module? This check alone probably doesn't make sense on a Unity site - and could potentially conflict if another site has a content type with that id - but for now this should work okay.

Reply:
Yeah, I think health conditions are an edge case